src/pages/api/loadQuestionList.ts

```diff
@@ -5,16 +5,35 @@ import { ID } from 'src/types/common'
 export interface LoadQuestionListParams {
   cid: ID
   topic?: string
+  page: number
 }
 
 export interface LoadQuestionListResults {
   problemList: QStem[]
+  maxPages: number
 }
 
-export default apiController<LoadQuestionListParams, LoadQuestionListResults>(async ({ cid, topic = '' }) => {
-  const qStems = await QStemModel.find({ class: { $eq: cid }, learningObjective: { $regex: topic, $options: 'i' } })
+export default apiController<LoadQuestionListParams, LoadQuestionListResults>(async ({ cid, topic = '', page }) => {
+  const questionsPerPage = 10
```

Comment:
I think this should be from API requesters

Reply:
> I think this should be from API requesters

Good idea updated it here https://github.com/kixlab/kuiz/pull/79/commits/f55e163fb1919631878316804080b909ba12b632 :)